cm_ovirt_vm_creator.py

```diff
@@ -77,7 +75,7 @@ def do_work(args):
         for idx in range(1, args.nodes+1):
             cluster_nodes.append(vm_name_template.format(name_prefix=args.name_prefix,
                                                          node_type="compute", i=idx))
-        print (cluster_nodes)
+        print(cluster_nodes, file=sys.stderr)
```

Comment:
why stderr?

Reply:
so stdout outputs something that you can save to a file and later `source` in bash, and all other human-friendly, not bash-compatible output will be in stderr.

Reply:
slight abuse of stderr. You could pass a file name to cm_ovirt_vm_creator and print the out to that file.